sdk/keyvault/azure-keyvault-administration/tests/_test_case.py

```diff
@@ -84,23 +84,8 @@ def setUp(self, *args, **kwargs):
             self.container_uri = "https://{}.blob.{}/{}".format(storage_name, storage_endpoint_suffix, container_name)
             self._scrub_url(real_url=self.container_uri, playback_url=container_playback_uri)
 
-            storage_account_key = os.environ.get("BLOB_PRIMARY_STORAGE_ACCOUNT_KEY")
```

Comment:
Should we remove this environment variable "BLOB_PRIMARY_STORAGE_ACCOUNT_KEY" in `test-resources,json` since here is the only place using it?